docs/UserGuide.md

```diff
@@ -529,9 +530,6 @@ Format: `modules del/INDEX`
 * The index **must be a positive integer** 1, 2, 3, …
 * Warning: This action is irreversible!
 
-Examples:
-* `modules del/1` - Deletes the 1st module in the modules list.
-
 1. You enter a command into the command box to delete a module, let's say module number 9 that we created previously.
 ![delete module before](images/moduletab_deletemodule_before.png)
 Figure 19: TR4CKER before executing `modules del/9`
```

Comment:
Seems to have multiple figures labelled as figure 19?

Reply:
i think ltr then standardise the figures bah